ex03.ex

```diff
@@ -77,7 +86,13 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+  def list_contains = fn ([h | t], givenValue) ->
```

Comment:
## -2

I think using `if` hurt you here—pattern matching makes it easier:

~~~ elixir
  def list_equal([], []),                              do: true
  def list_equal([ h | t1 ], [h | t2]),                do: list_equal(t1, t2)
  def list_equal(a, b) when is_list(a) and is_list(b), do: false
~~~